frontend/src/pages/pipelines/global/modelCustomization/startRunModal/StartRunModal.tsx

```diff
@@ -33,78 +44,98 @@ const StartRunModal: React.FC<StartRunModalProps> = ({ onSubmit, onCancel }) =>
           isSubmitDisabled={!canContinue}
         />
       }
-      variant="medium"
+      variant={ModalVariant.medium}
       data-testid="start-run-modal"
+      hasNoBodyWrapper
     >
-      <Form>
-        <Stack hasGutter>
-          <StackItem>
-            Fine-tune your models to improve their performance, accuracy, and task specialization,
-            using the{' '}
-            <Button
-              data-testid="lab-method"
-              variant="link"
-              isInline
-              component="a"
-              style={{ textDecoration: 'none' }}
-              onClick={() => {
-                // TODO: Link to documentation
-              }}
-            >
-              LAB method
-            </Button>
-            . Before creating a run, a taxonomy is needed and a teacher and judge model must be
-            configured.{' '}
-            <Button
-              data-testid="learn-more-prerequisites"
-              variant="link"
-              isInline
-              component="a"
-              style={{ textDecoration: 'none' }}
-              onClick={() => {
-                // TODO: Link to documentation
-              }}
-            >
-              Learn more about prerequisites for InstructLab fine-tuning
-            </Button>
-            .
-          </StackItem>
-          <StackItem>
-            <FormGroup
-              label="Data science project"
-              fieldId="start-run-modal-project-name"
-              isRequired
-            >
-              <Stack hasGutter>
+      <ModalHeader
+        title="Start a LAB-tuning run"
+        description={
+          <Stack hasGutter>
```

Comment:
No need to use a `Stack` component here IMO, a `<br />` between the first sentence and the second link is enough.

<img width="846" alt="Screenshot 2025-03-11 at 11 18 56 AM" src="https://github.com/user-attachments/assets/853c00e2-770d-44a9-822d-e8c8d34033f9" />